docs/api-docs/apps/tutorials/sample-app-next/04-step-2-connect-to-bc.md

```diff
@@ -0,0 +1,296 @@
+[Step 1: Set up your local environment](../docs/Step-1-Set-up-your-local-environment.md)
+
+# Step 2: Connect your app to BigCommerce
+
+In this step, you will connect your app to the BigCommerce ecosystem embedding it into **Draft Apps**.
+
+## Install node-bigcommerce
+
+To authenticate and use your app with the BigCommerce API, install the [node-bigcommerce](https://www.npmjs.com/package/node-bigcommerce) node module.
+
+```shell
+npm install node-bigcommerce
+```
+
+## Setup the auth lib page
+
+1. In the root directory of your app, create a `lib` folder.
+
+2. In the `lib` folder, create an `auth.ts` file.
+
+3. Open the `auth.ts` file and add the `BigCommerce` import at the top of the file.
+
+```js
+import * as BigCommerce from 'node-bigcommerce';
+```
+
+4. Create a BigCommerce instance required as part of the authorization step when first installing the application.
+
+```js
+// Create BigCommerce instance
+// https://github.com/getconversio/node-bigcommerce
+const bigcommerce = new BigCommerce({
+    logLevel: 'info',
+    clientId: process.env.CLIENT_ID,
+    secret: process.env.CLIENT_SECRET,
+    callback: process.env.AUTH_CALLBACK,
+    responseType: 'json',
+    headers: { 'Accept-Encoding': '*' },
+    apiVersion: 'v3'
+});
+
+const bigcommerceSigned = new BigCommerce({
+    secret: process.env.CLIENT_SECRET,
+    responseType: 'json'
+});
+
+interface QueryParams {
+    [key: string]: string;
+}
+```
+
+The `bigcommerceSigned` function will be called when loading or uninstalling the application.
+
+5. Export the `getBCAuth` function.
+
+```js
+export function getBCAuth(query: QueryParams) {
+    return bigcommerce.authorize(query);
+}
+```
+
+The `authorize` method will retrieve your permanent access token and return it in the `data` object.
```

Comment:
we might want to mention that the `authorize` method is used for the `auth` API endpoint (gets called when installing/ updating the app).  err since below we mention `verify` syncs to `load` and `uninstall` endpoints